pkg/asset/targets/targets.go

```diff
@@ -28,6 +29,7 @@ var (
 		&machines.Worker{},
 		&manifests.Manifests{},
 		&manifests.Openshift{},
+		&clusterapi.Cluster{},
```

Comment:
need to drop this before merging. I wonder if this could be causing the unit test to fail.  This will be handled by https://github.com/openshift/installer/pull/8166 which will hopefully merge soon

Reply:
Removed.